IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcCourse/Documentation.md

```diff
@@ -4,3 +4,22 @@ Structurally a course does not have capacity to carry loads over open span, or t
 * Graded sand layers
 * Cement bounded material (CBM)
 * Asphalt layers
+
+NOTE For road applications, the following types of courses were identified but not added to the schema model due to the fact that the elements (including terms and definitions) can vary greatly between nations; instead, these would be typical values of the inherited attribute ObjectType:
+
+- **Paving** : Type of Course directly in contact with traffic loads e.g. in gravel road (also Surface layer or Wearing course).
+- **AnticapillaryLayer** : Type of Course, layer preventing capillary rise of water from underlying layers. Definition from PIARC: Protective bottom layer with no fines, designed to prevent capillary rise of water from the subgrade soil or from underlying layers.
```

Comment:
```suggestion
- **AnticapillaryLayer** : Type of Course, layer preventing capillary rise of water from underlying layers. 
>NOTE Definition according to PIARC: Protective bottom layer with no fines, designed to prevent capillary rise of water from the subgrade soil or from underlying layers.
```